src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -35,7 +35,10 @@ export class DocService {
       dashboard: `${domain}mgr/dashboard`,
       grafana: `${domain}mgr/dashboard/#enabling-the-embedding-of-grafana-dashboards`,
       orch: `${domain}mgr/orchestrator`,
-      pgs: `http://ceph.com/pgcalc`
+      pgs: `http://ceph.com/pgcalc`,
+      help: `https://ceph.io/help/`,
+      security: `https://ceph.io/security/`,
+      trademarks: `https://ceph.io/legal-page/trademarks/`
```

Comment:
We should use the same domain for all links.
Not sure which one is the recommended one.

Reply:
ceph.io is the preferred one.

Reply:
The official sites, even the ceph.com redirects to ceph.io, so I think we should use ceph.io as the common domain for all links.

Reply:
Just reloaded and saw @LenzGr 's earlier comment. Going with ceph.io.

Reply:
Done.